integration-cli/docker_cli_build_test.go

```diff
@@ -863,10 +862,10 @@ func (s *DockerSuite) TestBuildCopyWildcard(c *check.C) {
 	ctx, err := fakeContext(fmt.Sprintf(`FROM busybox
 	COPY file*.txt /tmp/
 	RUN ls /tmp/file1.txt /tmp/file2.txt
-	RUN mkdir /tmp1
+	RUN sh -c "mkdir /tmp1"
```

Comment:
Hum, I would think that using the JSON syntax would make more sence then (it's more clear than using `sh -c` in this case).